README.md

```diff
@@ -147,3 +147,42 @@ To debug/test changes made in the `InstallIgnition` asset, follow the steps desc
 * RecoveryIgnition - the ignition config embedded into the recovery ISO
 
 ## High-Level Flow
+
+The appliance build process consists of the following steps:
+1. Download CoreOS ISO
+   * Extracted from the `machine-os-images` image (included in the release payload)
+   * Used as the base image of the recovery ISO
+2. Generate recovery CoreOS ISO
+   * Generated by embedding a custom bootstrap ignition to the base CoreOS ISO
```

Comment:
might be a good addition to specify the partition name

Reply:
right, added